library/src/main/java/com/pokegoapi/api/inventory/CandyJar.java

```diff
@@ -66,14 +50,16 @@ public void addCandy(PokemonFamilyId family, int amount) {
 	 * @param amount Amount of candies to remove
 	 */
 	public void removeCandy(PokemonFamilyId family, int amount) {
-		if (candies.containsKey(family)) {
-			if (candies.get(family) - amount < 0) {
-				candies.put(family, 0);
+		synchronized (candies) {
```

Comment:
i don't think that you can, it's a delta value. you haven't to put something you have to get the current val, subtract and put.
you can use compute but it's jdk8 only